front/src/modules/ui/relation-picker/components/SingleEntitySelectBase.tsx

```diff
@@ -36,8 +37,15 @@ export function SingleEntitySelectBase<
 }) {
   const containerRef = useRef<HTMLDivElement>(null);
   const entitiesInDropdown = isDefined(entities.selectedEntity)
```

Comment:
Nice!
Could you apply 
```
.filter((entity) =>
          isNonEmptyString(entity.name) 
```
on the whole `entitiesInDropdown` instead of applying to both  `entities.selectedEntity` and `entities.entitiesToSelect`.

We also have the same issue in `MultipleEntitySelect`

Reply:
> Nice! Could you apply
> 
> ```
> .filter((entity) =>
>           isNonEmptyString(entity.name) 
> ```
> 
> on the whole `entitiesInDropdown` instead of applying to both `entities.selectedEntity` and `entities.entitiesToSelect`.
> 
> We also have the same issue in `MultipleEntitySelect`

fix sent.